src/transformers/modeling_bert.py

```diff
@@ -223,6 +226,11 @@ def __init__(self, config):
 
         self.dropout = nn.Dropout(config.attention_probs_dropout_prob)
 
+        self.position_embedding_config = config.position_embedding_config
+        if self.position_embedding_config > 0:
+            self.max_position_embeddings = config.max_position_embeddings
```

Comment:
no need to save `max_position_embeddings` here IMO -> you can use `self.config.max_position_embeddings` further down.

Reply:
max_position_embeddings is initialized here and it is used in line 273, where the self.config or config is not available in forward().